roles/rsyslog/defaults/main.yaml

```diff
@@ -959,7 +959,7 @@ rsyslog_conf_viaq_elasticsearch:
                   set $.viaq_index_name = exec_template("prefix_index_template");
               }
           {% for res in rsyslog_elasticsearch %}
-          {% if res.index_prefix|default("") == "" %}
+          {% if loop.length == 1 %}
```

Comment:
Can you please explain what is this loop.length ==1 used for?  I'm probably missing something in the logic ... @nhosoi